src/app.js

```diff
@@ -28,6 +30,32 @@ export function createApp() {
     app.use(require('koa-logger')());
   }
 
+  app.use(
+    reportTo([
+      {
+        group: 'cspreport',
+        maxAge: 365 * 24 * 60 * 60,
+        endpoints: [{ url: '/__cspreport__' }],
+      },
+    ])
+  );
+  app.use(
+    helmet({
+      contentSecurityPolicy: {
+        directives: {
+          defaultSrc: ["'none'"],
```

Comment:
I feel like this section could use a few comments explaining why the values were chosen. I think it could be as simple as something like "this is an API and shouldn't be loaded as a webpage."

Reply:
yeah I can do that; actually I just used the CSP proposed in https://github.com/firefox-devtools/profiler-server/issues/18 :)

Reply:
yeah I can do that; actually I just used the CSP proposed in #18 :)

Reply:
I added a link to https://github.com/mozilla-services/websec-check